### PR TITLE
Id 221 ship create api skeleton for admin UI

### PR DESCRIPTION
**Pull request by martinsaporiti**
## Notes
* New admin server ()
* Docker-compose for the admin server
* makefile updated

### CODE REVIEW

File: internal/api_admin/server.go [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+// Server implements StrictServerInterface and holds the implementation of all API controllers
+// This is the glue to the API autogenerated code
+type Server struct {
+	cfg              *config.Configuration
+	identityService  ports.IdentityService
+	claimService     ports.ClaimsService
+	schemaService    ports.SchemaService
+	publisherGateway ports.Publisher
+	packageManager   *iden3comm.PackageManager
+	health           *health.Status
```
Comment (x1m3) [inline internal/api_admin/server.go:25]: Not sure if we are gonna need all of these.  But it is ok.

File: internal/api_admin/errors.go [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+	switch err.(type) {
+	case AuthError:
+		w.WriteHeader(http.StatusUnauthorized)
+		w.Header().Add("WWW-Authenticate", `Basic realm="restricted", charset="UTF-8"`)
+		_, _ = w.Write([]byte("\"Unauthorized\""))
+	default:
+		w.WriteHeader(http.StatusInternalServerError)
+		_, _ = w.Write([]byte(err.Error()))
+	}
+}
```
Comment (x1m3) [inline internal/api_admin/errors.go:33]: This code is repeated. Don't you think we could move the original one to an internal package and then import in both sides?

**Review by x1m3 — CHANGES_REQUESTED** [review]

**Comment by x1m3** [comment]
LGTM. Just added a request change to unify the middleware, but consider it more a suggestion than a request change. It if it not easy to move, let's go on and we iterate over it later.

File: internal/api_admin/server.go (hunk 1 above)
Comment (martinsaporiti) [reply internal/api_admin/server.go:25]: Well, me neither too. I left those parameters for now until we will have more information regarding what we are going to need.

**Review by x1m3 — APPROVED** [review]

**Review by javip97 — APPROVED** [review]
LGTM

**Pull request merged**